SortingWithSOLID/src/Logger/Logger.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "./Duration/Duration.h"
+#include <Windows.h>
+#include <iostream>
+#include <limits>
+#include <map>
+
+void printDuration(const std::multimap<Duration, std::string> durations)
+{
+	const std::string fastestSortingName = getFastestSortingName(durations);
+	std::cout << "SortingName    -->   Duration(h : min : sec) \n";
+	for (const auto& dur : durations)
```

Comment:
There you can improve readability. For that, you can use [structured_binding](https://en.cppreference.com/w/cpp/language/structured_binding)
```suggestion
	for (const auto [duration, name] : durations)
```